cmd/proxy/actions/app_proxy.go

```diff
@@ -21,6 +21,7 @@ func addProxyRoutes(
 ) error {
 	app.GET("/", proxyHomeHandler)
 	app.GET("/healthz", healthHandler)
+	app.GET("/readiness", getReadinessHandler(s))
```

Comment:
can we make it `/readyz` to stick with convention?

Reply:
sure, why not 👍